deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/config/ZKMysqlDataSourceFactory.java

```diff
@@ -70,8 +64,8 @@ public void setPool(String pool) {
     public BasicDataSource build() throws Exception {
         // get from local ZK maintained DB config file
         DBConfigReader reader = new DBConfigReader();
-        String host = this.proxyHost != null ? this.proxyHost : reader.getHost(replicaSet);
-        int port = this.proxyPort != null ? Integer.parseInt(this.proxyPort) : reader.getPort(replicaSet);
+        String host = reader.getHost(replicaSet);
```

Comment:
i think it is still good to keep the functionality to support proxy

Reply:
discussed on slack for the reason behind cleaning up.